synapse/config/cache.py

```diff
@@ -148,11 +148,15 @@ def generate_config_section(self, **kwargs) -> str:
           per_cache_factors:
             #get_users_who_share_room_with_user: 2.0
 
-          # Controls how long an entry can be in a cache without having been
-          # accessed before being evicted. Defaults to None, which means
-          # entries are never evicted based on time.
+          # Controls whether cache entries are evicted after a specified time
+          # period. Defaults to true.
+          # expire_caches: false
```

Comment:
```suggestion
          # Controls whether cache entries are evicted after a specified time
          # period. Defaults to true.
          #
          # expire_caches: false
```

Reply:
This block should also have one of those "Uncomment to disable this feature" sentences.